tests/components/test_alert.py

```diff
@@ -1,24 +1,29 @@
 """The tests for the Alert component."""
+import unittest
 # pylint: disable=protected-access
 from copy import deepcopy
-import unittest
 
-from homeassistant.setup import setup_component
-from homeassistant.core import callback
-from homeassistant.components.alert import DOMAIN
 import homeassistant.components.alert as alert
 import homeassistant.components.notify as notify
+from homeassistant.components.alert import DOMAIN
 from homeassistant.const import (
     ATTR_ENTITY_ID, CONF_ENTITY_ID, STATE_IDLE, CONF_NAME, CONF_STATE,
     SERVICE_TOGGLE, SERVICE_TURN_OFF, SERVICE_TURN_ON, STATE_ON, STATE_OFF)
-
+from homeassistant.core import callback
+from homeassistant.setup import setup_component
 from tests.common import get_test_home_assistant
 
 NAME = "alert_test"
 DONE_MESSAGE = "alert_gone"
 NOTIFIER = 'test'
 TEMPLATE = "{{ states.sensor.test.entity_id }}"
 TEST_ENTITY = "sensor.test"
+TITLE = 'test'
```

Comment:
Shouldn't we test that we can render the title template, by using a template?

Reply:
Agreed. I've changed `TITLE` into a template.